interface/forms/soap/C_FormSOAP.class.php

```diff
@@ -1,40 +1,44 @@
 <?php
 
-/**
+/*
  * soap form
- *
  * @package   OpenEMR
  * @link      http://www.open-emr.org
  * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @author    Sherwin Gaddis <sherwingaddis@gmail.com>
  * @copyright Copyright (c) 2019 Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2022 Sherwin Gaddis <sherwingaddis@gmail.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
 require_once($GLOBALS['fileroot'] . "/library/forms.inc.php");
 require_once("FormSOAP.class.php");
+require_once("Bootstrap.php");
 
 use OpenEMR\Common\Csrf\CsrfUtils;
 
 class C_FormSOAP extends Controller
```

Comment:
Can the `extends Controller` now be remove?

Reply:
I will find out..

Reply:
No, here is my theory on why. The FormSOAP.class.php file still depends on the parent::populate(). @adunsulag would need to confirm that. I have been far enough down that path of how the populate works. It reads the table columns. Using that list of columns it auto matches the column to the getter and setter method. Depending on the mod as the trigger. Until the dependency on the populate method. The extends controller has to stay.

Reply:
What @juggernautsei is saying is correct.  Still have to do the extends here.